src/System.Runtime.InteropServices.RuntimeInformation/src/Architecture.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public enum Architecture
```

Comment:
Scratch the last part - I see there is *one more* enum type for architecture - `ProcessorArchitecture` - still the question why you can't have just one definition in managed code stands.